data/services/volume-services.toml

```diff
@@ -0,0 +1,5 @@
+# Service metadata lives
```

Comment:
Metadata lives at https://github.com/cloudfoundry/nfsbroker/blob/7b95addb11f6f71fd207f9ed69a935b4de97ba34/nfsbroker/nfsbroker.go#L91-L125. We only expose the first service and plan.

Reply:
@jmcarp I'm not sure how to read this - what's the service name for the first one? It seems to just say `b.static.ServiceName`. And to verify, the description is "Existing NFSv3 volumes (see: https://code.cloudfoundry.org/nfs-volume-release/)"?

Reply:
Er, sorry, the service name comes from another place, for some reason: https://github.com/18F/cg-deploy-volume-services/blob/7fbc82688c7d73930ff113db5a8de0886f73ff80/ci/manifest.yml#L6. So the service name is `nfs`, the plan is `Existing`, and you got the description.